yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -321,14 +327,38 @@ func setDefaultForNullString() {
 }
 
 // escaping single quote character
+// checks and process the given string is a single byte character
+func parseAndCheckSingleByteChar(value string) (string, bool) {
+	if len(value) == 1 {
+		return value, true
+	}
+	parsedValue, err := strconv.Unquote(`"` + value + `"`)
+	if err != nil || len(parsedValue) != 1 {
+		return parsedValue, false
+	}
+	return parsedValue, true
+}
+
+// in case of csv file format, escape and quote characters are required to be escaped with
+//
+//	backslash if they are single quote or backslash with E in copy Command
```

Comment:
I feel a little confused in this statement `if they are single quote or backslash with E in copy Command` 
will this be better:
`if they are single quote or backslash`
or
`if they are single quote or backslash provided E in copy Command`

what do you think?